app/views/handlers/content.py

```diff
@@ -23,7 +23,9 @@ def rendered_block(self):
         ) or self._get_content_title(transformed_block)
         self._set_page_title(content_page_title)
         return self.placeholder_renderer.render(
-            transformed_block, self._current_location.list_item_id
+            transformed_block,
```

Comment:
Since you use named attributes when you invoke this method in `list_collector_block.py` I would suggest using named attributes here as well.

Reply:
Updated this 👍